addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -129,8 +129,8 @@ export class MassMailingHtmlField extends HtmlField {
             // Wait for the css and images to be loaded.
             await iframePromise;
             const editableClone = iframe.contentDocument.querySelector('.note-editable');
-            this.cssRules = this.cssRules || getCSSRules($editable[0].ownerDocument);
-            await toInline($(editableClone), this.cssRules, $(iframe));
+            const cssRules = this.cssRules || getCSSRules($editable[0].ownerDocument);
+            await toInline($(editableClone), cssRules, $(iframe));
```

Comment:
This completely disables the cache, thus greatly slowing down the operations of mass_mailing that trigger a convert inline. I quickly told @Zynton about the issue and he suggested to invalidate the cache when something is changed in the design tab. That would be a better way to fix the issue so that we can keep the caching feature.

Reply:
@dmo-odoo  , I changed the fix , can you take a look ? Thanks

Reply:
Unless I'm mistaken, I don't think this is what @Zynton had in mind. Maybe he can chime in?

Reply:
Indeed, I meant to just invalidate the cache whenever a change is made in the design tab.